src/solver/conv_mlir_igemm_fwd_xdlops.cpp

```diff
@@ -126,8 +126,14 @@ bool PerformanceConvMlirIgemmXdlops::IsValid(const ConvolutionContext& ctx) cons
 
 bool PerformanceConvMlirIgemmXdlops::SetNextValue(const ConvolutionContext& /*config*/)
 {
+    static bool allTuningExercised = false;
```

Comment:
Ditto.

Reply:
Done in [478fdb9](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1435/commits/478fdb99e531c3a11c0eada133b367bddc8ebcda)